drivers/intel/soc/stratix10/io/s10_memmap.c

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (c) 2014-2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <assert.h>
+#include <common/debug.h>
+#include <io_driver.h>
+#include <io_memmap.h>
+#include <io_storage.h>
+#include <platform_def.h>
+#include <string.h>
+#include <utils.h>
+
+#include "../plat/intel/soc/stratix10/drivers/qspi/cadence_qspi.h"
```

Comment:
This include doesn't look great. I guess this starts the search in ``include/``, goes down one folder and then moves around the codebase. It doesn't look like a clean solution.

Reply:
I had changed the path relative to plat/intel/soc/stratix10. Thanks